### PR TITLE
Use `rlang::caller_arg()` for default arg in checkers

### LINKED ISSUE #49: Use rlang calls in checkers
Use all the nice caller arg stuff: https://rlang.r-lib.org/reference/topic-error-call.html

### PR DESCRIPTION
**Pull request by zsusswein**
As detailed in:
https://rlang.r-lib.org/reference/topic-error-call.html#input-checkers-and-caller-arg-

Also standardized the formatting of the calls to have a newline after each arg.

Closes #49

### CODE REVIEW

**Comment by zsusswein** [comment]
Marking as draft until #52 merges.

File: R/checkers.R
Comment (seabbs) [inline R/checkers.R:292]: minor but this function has different indenting of its args to above

**Review by seabbs — APPROVED** [review]
Nice. LGTM. Not clear if failing checks were due to being blocked by the other PR or indicating something not working as expected

**Pull request merged**
